tsconfig.json

```diff
@@ -0,0 +1,26 @@
+{
+  "compilerOptions": {
+    "module": "commonjs",
+    "lib": ["dom", "es2017"],
+    "target": "es6",
+    "declaration": true,
+    "noImplicitAny": true,
```

Comment:
It's safer to use `"strict": true` so the TS compiler performs null checks and help catch other possible runtime issues.